ports/atmel-samd/mpconfigport.h

```diff
@@ -42,6 +42,7 @@
 #define CIRCUITPY_MCU_FAMILY                        samd21
 #define MICROPY_PY_SYS_PLATFORM                     "Atmel SAMD21"
 #define SPI_FLASH_MAX_BAUDRATE 8000000
+#define MICROPY_ERROR_REPORTING (MICROPY_ERROR_REPORTING_TERSE)
```

Comment:
I don't want to switch to errors to TERSE. Good error messages are critical when beginners are beginning to learn to code. It is not something we should sacrifice to have dynamic EInk rotation.

Reply:
OK, I think I found enough small items to make it fit.